core/src/test/scala/unit/kafka/admin/TopicCommandWithZKClientTest.scala

```diff
@@ -31,7 +31,7 @@ import org.scalatest.Assertions.intercept
 
 import scala.util.Random
 
-class TopicCommandTest extends ZooKeeperTestHarness with Logging with RackAwareTest {
+final class TopicCommandWithZKClientTest extends ZooKeeperTestHarness with Logging with RackAwareTest {
```

Comment:
I forgot to rename class with the previous `git mv`.

Reply:
Why did we make it final?

Reply:
Habit. Do we want developers extending test suites?

Reply:
No, but we don't actively protect against it. I think nearly all tests don't have `final`. It's one of those things that looks a bit weird if we do it occasionally but it's not actively recommended.

Reply:
Done.